packages/market-service/src/utils/getRatelimitedAxios.ts

```diff
@@ -0,0 +1,6 @@
+import axios from 'axios'
+import rateLimit from 'axios-rate-limit'
+
+export const getRatelimitedAxios = (rps: number, perMilliseconds = 1000) => {
+  return rateLimit(axios.create(), { maxRequests: rps, perMilliseconds })
+}
```

Comment:
```suggestion
export const getRateLimitedAxios = (maxRPS: number) => rateLimit(axios.create()).setMaxRPS(maxRPS)
```

please read the docs carefully https://github.com/aishek/axios-rate-limit#usage

```
// note maxRPS is a shorthand for perMilliseconds: 1000, and it takes precedence
// if specified both with maxRequests and perMilliseconds
```

Reply:
yeah that's why I removed `maxRPS` in favor of more granular control using `maxRequests` and `perMilliseconds`